packages/api/src/controllers/usage.ts

```diff
@@ -63,6 +65,68 @@ export const calculateOverUsage = async (product, usage) => {
   return overUsage;
 };
 
+export const getUsagePercentageOfLimit = async (product, usage) => {
+  let limits: any = {};
```

Comment:
ts to the rescue (from `any`) 😆 
```suggestion
  let limits: Record<string, number> = {};
```